csrankings-1.csv

```diff
@@ -2009,7 +2009,7 @@ Danny Raz,Technion,http://www.cs.technion.ac.il/~danny,vLzruXMAAAAJ
 Danny Weyns,KU Leuven,https://distrinet.cs.kuleuven.be/people/danny,MG8iJPAAAAAJ
 Danqi Chen,Princeton University,https://www.cs.princeton.edu/~danqic,sVR8ktkAAAAJ
 Dante Augusto Couto Barone,UFRGS,http://www.inf.ufrgs.br/site/docente/dante-augusto-couto-barone,O4nTWhwAAAAJ
-Danupon Nanongkai,KTH Royal Institute of Technology,http://www.csc.kth.se/~danupon,ofNbQXAAAAAJ
+Danupon Nanongkai,University of Copenhagen,http://www.csc.kth.se/~danupon,ofNbQXAAAAAJ

```

Comment:
No indication on home page(s) that this faculty member has moved - please point to a site indicating such.

Reply:
Confirmed:
https://research.ku.dk/search/result/?pure=en%2Fpersons%2Fdanupon-na-nongkai(458ed2ce-6558-42df-b49b-14628304d786).html
https://barc.ku.dk/news/danupon-nanongkai-joins-barc/